frontend/server/controllers/NotificationController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * BadgesController
+ *
+ * @author carlosabcs
+ */
+class NotificationController extends Controller {
+    /**
+     * Returns a list of unread notifications for user
+     *
+     * @param Request $r
+     * @return array
+     * @throws InvalidDatabaseOperationException
+     */
+    public static function apiMyList(Request $r) {
+        self::authenticateRequest($r);
+        return [
+            'status' => 'ok',
+            'notifications' => NotificationsDAO::getUnreadNotifications($r->user),
```

Comment:
```suggestion
            'notifications' => is_null($r->user) ?
                [] :
                NotificationsDAO::getUnreadNotifications($r->user),
```

si haces login como identidad, no debería arrojar una excepción de que le estás pasando `null` a un método que no espera `null`.